### ISSUE TITLE
safeRequire returning undefined even when module is installed

### ISSUE THREAD

**Issue by ritch** [strongloop/loopback#833]
We're running into an issue where the following setup is causing 
[https://github.com/strongloop/loopback/blob/master/lib/express-middleware.js#L8](safeRequire) to return undefined.

```
arc depends on loopback
arc depends on workspace
workspace depends on loopback
workspace depends on cookie-parser
```

Since the versions are compatible (I assume) `npm` installs only one `loopback` package in arc's node_modules directory (and nothing in workspace). This breaks the assumptions `safeRequire()` makes - which is that the `loopback` will be a dependency of the module that has the independent middleware installed (eg. serve-favicon, which is installed in workspace).

I'm not exactly sure how we can fix this with npm's behavior. Npm chooses to install only a single loopback module. Currently we just install the modules in arc. This is a bit unsettling because they are not arc's dependencies (they are workspace's).

**Comment by bajtos** [comment]
I believe this issue is a duplicate of https://github.com/strongloop/loopback-boot/issues/73

**Comment by bajtos** [comment]
This should be fixed as of `loopback-boot@2.7.0`. Please upgrade your projects to use this new version and reopen this issue if the problem still persists.
